core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/debug/BreakpointManagerImpl.java

```diff
@@ -23,18 +24,9 @@
 import org.eclipse.che.ide.api.project.tree.VirtualFile;
 import org.eclipse.che.ide.api.project.tree.VirtualFileImpl;
 import org.eclipse.che.ide.api.project.tree.VirtualFileInfo;
-import org.eclipse.che.ide.debug.Breakpoint;
+import org.eclipse.che.ide.debug.*;
```

Comment:
Wildcard imports are not allowed in our code
https://github.com/codenvy/codenvy-dev-resources/wiki/Source-file#imports